tox.ini

```diff
@@ -15,3 +16,5 @@ commands =
   flake8 xsnippet_api/ tests/
 # check that unittests are passed
   py.test --cov xsnippet_api/ --cov-append tests/
+# check the swagger definition of the API
+  swagger-flex -s contrib/swagger.yaml
```

Comment:
We only need it because the package and all its dependencies are installed into a virtual environment before running any commands.

I didn't know about ``skip_install = True`` - that will help, thanks!